src/js/services/verifyByThirdPartyService.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+angular.module('copayApp.services').factory('verifyByThirdPartyService', function verifyByThirdPartyServiceFactory($log, $http, lodash, configService) {
+
+  var root = {};
+
+  root.getTx = function(service, txid, cb) {
+    return explorers[service.name].getTx(txid, cb);
+  };
+
+  var getTxFromBlockchainInfo = function(txid, cb) {
+    var service = lodash.find(configService.getDefaults().blockExplorerServices, function(s) {
+      return (s.name == 'Blockchain.info');
+    });
+
+    requestTx(service.name, service.url + txid, function(err, resp) {
+      if (err) return cb(err);
+      if (resp.notFound) return cb(null, resp);
+
+      var params = {};
+      params.name = service.name;
+      params.notFound = resp.status == 404;
+      params.success = resp.status == 200;
+      params.out = [];
+      lodash.each(resp.data.out, function(out) {
+        params.out.push({
+          address: out.addr,
+          amount: parseInt(out.value)
+        });
+      });
+      return cb(null, params);
+    });
+  };
+
+  var getTxFromBlockrIo = function(txid, cb) {
+    var service = lodash.find(configService.getDefaults().blockExplorerServices, function(s) {
+      return (s.name == 'Blockr.io');
+    });
+    requestTx(service.name, service.url + txid, function(err, resp) {
+      if (err) return cb(err);
+      if (resp.notFound) return cb(null, resp);
+
+      var settings = configService.getSync().wallet.settings;
+      var unitToSatoshi = settings.unitToSatoshi;
+      var params = {};
+      params.name = service.name;
+      params.notFound = resp.status == 404;
+      params.success = resp.status == 200;
+      params.out = [];
+      lodash.each(resp.data.data.vouts, function(out) {
+        params.out.push({
+          address: out.address,
+          amount: parseInt(out.amount * unitToSatoshi)
+        });
+      });
+      return cb(null, params);
+    });
+  };
+
+  var explorers = {
```

Comment:
here there shoudl be all the info about the services:


name,
url,
parser: function(response) {
  return normalizedJason;
}